pkgs/development/python-modules/ultralytics/default.nix

```diff
@@ -0,0 +1,109 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+
+  # build-system
+  setuptools,
+
+  # dependencies
+  scipy,
+  matplotlib,
+  opencv-python,
+  pillow,
+  pyyaml,
+  requests,
+  torch,
+  torchvision,
+  tqdm,
+  psutil,
+  py-cpuinfo,
+  pandas,
+  seaborn,
+  ultralytics-thop,
+
+  # tests
+  pytestCheckHook,
+  onnx,
+  onnxruntime,
+}:
+
+buildPythonPackage rec {
+  pname = "ultralytics";
+  version = "8.3.85";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ultralytics";
+    repo = pname;
```

Comment:
use of pname like this is an antipattern as it breaks overrides

Reply:
fixed